Rakefile

```diff
@@ -2,20 +2,10 @@ require "bundler/gem_tasks"
 
 require 'rake/testtask'
 
-# Rake::TestTask.new do |t|
-#   t.libs << 'test'
-# end
-
-# desc "Run tests"
-# task :default => :test
-
-
-
-desc "Run the unit test suite"
-task :default => 'test:units'
+desc 'Run the unit test suite'
+task default: 'test:units'
```

Comment:
What do you think about having the default run the both unit and remote tests? Seems like there's a greater chance that folks won't run them otherwise.

Reply:
I'm torn on this, it's a good conversation to have. A concern when running both by default is that not all contributors (especially open source contributors) will have a full environment up and running. That all being said, folks would still be able to run just unit tests with `rake test:unit`. I'm cool with either way we decide but did want to highlight the open source angle.

Reply:
Fair point. It's probably better to keep the barrier to entry low here.